prime-router/src/main/kotlin/azure/WorkflowEngine.kt

```diff
@@ -850,6 +849,31 @@ class WorkflowEngine(
                 FileSettings("$baseDir/settings", orgExt = ext)
             }
         }
+
+        /**
+         * Always find tasks at least this old.  This covers for  extended downtime due to a crash,
+         * as well as for 25 hour days etc.
+         */
+        const val BATCH_LOOKBACK_PADDING_MINS: Long = 180 // 3 hours
+
```

Comment:
This padding is important.   The backstopTime is always at least this big.     In cases where there is an unrecoverable failure inside BatchFunction, we'll get exceptions for 3 hours for that receiver, until it moves on, which is a minor pain.   OTOH, we want the system to be able to go down for considerable times and self-heal upon restart.  Hence the 3 hours.  Note this is purposely NOT a function of the receiver batching frequency.

Note if the system went down for longer, we can easily and safely manually force batch TASKs to run by simply setting their next_action_at to now().